go.mod

```diff
@@ -1,57 +1,167 @@
 module github.com/redhat-developer/observability-operator/v3
 
-go 1.16
+go 1.18
 
 require (
 	github.com/ghodss/yaml v1.0.0
-	github.com/go-logr/logr v0.2.1
+	github.com/go-logr/logr v1.2.3
 	github.com/goccy/go-yaml v1.9.5
-	github.com/integr8ly/grafana-operator/v3 v3.10.3
-	github.com/onsi/ginkgo v1.14.0
-	github.com/onsi/gomega v1.10.3
+	github.com/grafana-operator/grafana-operator/v4 v4.5.0
+	github.com/onsi/ginkgo v1.16.5
+	github.com/onsi/gomega v1.20.1
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
-	github.com/operator-framework/api v0.3.20
-	github.com/operator-framework/operator-registry v1.12.6-0.20200611222234-275301b779f8
+	github.com/operator-framework/api v0.17.1
+	github.com/operator-framework/operator-registry v1.26.2
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus-operator/prometheus-operator v0.43.0
-	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.43.0
+	github.com/prometheus-operator/prometheus-operator v0.50.0
+	github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring v0.50.0
 	github.com/sirupsen/logrus v1.8.1
-	k8s.io/api v0.20.6
-	k8s.io/apimachinery v0.20.6
+	k8s.io/api v0.24.3
+	k8s.io/apimachinery v0.24.3
 	k8s.io/client-go v12.0.0+incompatible
```

Comment:
hey @vmanley; I think the client-go will also have to be bumped to 0.24.3

Reply:
@MStokluska Hey sorry missed this comment. That's no problem